index.html

```diff
@@ -551,6 +551,7 @@ <h4 class="modal-title">ScanCode Header Information</h4>
     </div><!-- /.modal -->
 
     <script>require('./assets/app/js/renderer.js');</script>
+    <!-- <script src='./assets/app/js/renderer.js'></script> -->
```

Comment:
Why add this as a comment?

Reply:
Actually, I had to experiment, add logs here and there, I removed most of the residue, but some code was left :/